api/test/unit/serializers/audio_serializers_test.py

```diff
@@ -0,0 +1,38 @@
+import uuid
+from unittest.mock import MagicMock
+
+from rest_framework.request import Request
+from rest_framework.test import APIRequestFactory
+
+import pytest
+
+from catalog.api.serializers.audio_serializers import AudioSerializer
+
+
+@pytest.fixture
+def req():
+    factory = APIRequestFactory()
+    request = factory.get("/")
+    request = Request(request)
+    return request
+
+
+@pytest.fixture
+@pytest.mark.django_db
```

Comment:
Is this mark necessary?

```suggestion
```

Reply:
I added this when the test used a Django instances and forgot to remove it. Removed in 8c4fd4b.